lib/commons/aria/roles.js

```diff
@@ -123,9 +122,8 @@ aria.implicitNodes = function (role) {
  * @memberof axe.commons.aria
  * @instance
  * @param  {HTMLElement} node The node to test
- * @return {Mixed}      Either the role or `null` if there is none
+ * @return {Mixed}      			Either the role or `null` if there is none
```

Comment:
Is all of this newly added whitespace intentional?

Reply:
got a bit creative with tabbing, so things are nicely tabulated/ lined up on an IDE. Nevertheless doesn't appeal on here.